core/src/main/scala/org/apache/spark/storage/BlockManagerMessages.scala

```diff
@@ -24,37 +24,37 @@ import org.apache.spark.util.Utils
 
 private[spark] object BlockManagerMessages {
   //////////////////////////////////////////////////////////////////////////////////
-  // Messages from the master to slaves.
+  // Messages from the master to storage endpoints.
   //////////////////////////////////////////////////////////////////////////////////
-  sealed trait ToBlockManagerSlave
+  sealed trait ToBlockManagerStorage
```

Comment:
Nit: I would prefer having `Endpoint` added because I don't think `Storage` is the correct noun.

Reply:
Sure